oncall/customRotation.txt

```diff
@@ -0,0 +1,4 @@
+#Enter a custom shift info in the following format:
```

Comment:
I'm in the boat that we shouldn't have this file in our repository as it will force us to have to create a PR any time we want to make custom changes to the rotation -- ideally we would be able to move this to a [repository variable](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository) that we can modify whenever we want independent of a PR.

Reply:
I took a look on the repo variable, and seems like it's viable only if either the repo is owned by us or the organization is owned by us. I believe we currently don't have access to change anything in the setting page.
Actually, if we have access to that page, we could also add tags to different issue templates for future automation use.